.changeset/tasty-pandas-cry.md

```diff
@@ -0,0 +1,5 @@
+---
+"@jspsych-contrib/plugin-html-choice": patch
```

Comment:
should this be a major release, or are you intending this to be version 0.1.1 of the plugin?